scripts/lag_keepalive.py

```diff
@@ -22,47 +29,64 @@ def log_error(msg):
     syslog.closelog()
 
 
-def sniff_lacpdu(device_mac, lag_member, lag_member_to_packet):
-    sniffed_packet = sniff(iface=lag_member,
-        filter="ether proto 0x8809 and ether src {}".format(device_mac),
-        count=1, timeout=30)
-    lag_member_to_packet[lag_member] = sniffed_packet
+def getCmdOutput(cmd):
+    proc = subprocess.Popen(cmd, stdout=subprocess.PIPE)
+    return proc.communicate()[0], proc.returncode
+
+
+def get_port_channel_config(portChannelName):
+    (processStdout, _) = getCmdOutput(["teamdctl", portChannelName, "state", "dump"])
+    return json.loads(processStdout)
+
+
+def craft_lacp_packet(portChannelConfig, portName):
+    portConfig = portChannelConfig["ports"][portName]
```

Comment:
What is a possibility of any of the following key accesses from the dict will fail with KeyError?

Reply:
Please add key presence check wherever needed, and default values (??) where keys are missing.

Reply:
There's a try-except in `main()` that catches everything and retries immediately.